server/api/helpers/encryption/encrypt.js

```diff
@@ -0,0 +1,21 @@
+import bcrypt from 'bcrypt';
```

Comment:
@Ibidapo I thought these functions where moved to the User model. Are there reasons why we still have them in this file ?

Reply:
This was an oversight. They're redundant given that hooks are handling these functionalities.